src/v2/api/group/GroupService.ts

```diff
@@ -69,4 +54,26 @@ export class GroupService {
 
     return results;
   }
+
+  async getCaptain(groupId: string) {
+    const students = await this.groupRepository.getStudents(groupId);
+    for (const student of students) {
+      const roles = await this.studentRepository.getRoles(student.userId);
+      for (const role of roles){
+        if (String(role) == "captain"){
```

Comment:
role is written in UPPER_CASE. Also you don't need to compare role with string, compare it with enum RoleName from prisma